project_task_dependency/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Can you please adopt the png image, according to the latest [readme template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)?